### ISSUE TITLE
A11Y - ShimmeredDetailsList columns not being read when sorted on click

### ISSUE THREAD

**Issue by achang0406** [OfficeDev/office-ui-fabric-react#10890]
### Environment Information

- **Package version(s)**: office-ui-fabric-react - 7.0.0
- **Browser and OS versions**: chrome Version 77.0.3865.90 (Official Build) (64-bit)

### Describe the issue:

Essentially when we click on a column to sort the data, the nvda fails to read the sorted column everytime.

NVDA can potentially successfully read when click to sort one column and potentiaily failed to do so in another column all within the same table using the shimmeredDetailsList component.

### Please provide a reproduction of the issue in a codepen:

We are actually **unsure** whether this is the the issue with shimmeredDetailsList component or our wrapper component that we are using (react-advanced-datatable, a published microsoft package).

We are filing this bug because we are seeing inconsistent behavior from one column to the next within the same table when used in our package. We are hoping that perhaps you can offer some wisdom to help determine the root of this issue.

#### Actual behavior:

Within the same table, nvda can fail to read the sorted state of a column when clicked but successfully do so in another column.

#### Expected behavior:

when click on a column to sort, the nvda should alert the user that the column is being sorted ascending or descending.

### Documentation describing expected behavior

NVDA and any other scan reader should alert the user the sorted state of any one column in a table when being sorted with a click.

@jgage176

**Comment by marygans** [comment]
Hi @achang0406 thanks for submitting this issue!

After taking a closer look, the [ W3 ARIA reference example](https://www.w3.org/TR/wai-aria-practices/examples/grid/dataGrids.html) also does not announce the sorting order and we are using `aria-sort` per the[ ARIA spec](https://www.w3.org/TR/wai-aria-1.1/#aria-sort). 

One alternative is, you can utilize the [Annouced](https://github.com/OfficeDev/office-ui-fabric-react/tree/master/packages/office-ui-fabric-react/src/components/Announced) component to get around and achieve this. We also have an[ example](https://developer.microsoft.com/en-us/fabric#/controls/web/detailslist/variablerowheights) utilizing Announced component. 
Thanks!
